src/Distributed/distributed_utils.jl

```diff
@@ -0,0 +1,99 @@
+using Oceananigans.Fields: AbstractField
+using Oceananigans.Grids:
+    interior_indices,
+    left_halo_indices, right_halo_indices,
+    underlying_left_halo_indices, underlying_right_halo_indices
+
+# TODO: Move to Grids/grid_utils.jl
+
+#####
+##### Viewing halos
+#####
+
+west_halo(f::AbstractField{LX, LY, LZ}; include_corners=true) where {LX, LY, LZ} =
+    include_corners ? view(f.data, left_halo_indices(LX, topology(f, 1), f.grid.Nx, f.grid.Hx), :, :) :
```

Comment:
what happens differently if include_corners is true or false?  having some trouble parsing it, sorry

Reply:
Ah so if you have 10x10 grid points and halos of size 1, then the halo region regions are of size 12x1 (or 1x12) but this includes the corners (so `include_corners = true`).

If `include_corners = false` then it'll return halos of size 10x1 (or 1x10).

Reply:
Thanks